istioctl/cmd/proxyconfig.go

```diff
@@ -824,30 +831,30 @@ func logCmd() *cobra.Command {
 		},
 		RunE: func(c *cobra.Command, args []string) error {
 			var err error
-			var loggerNames []string
+			loggerNames := map[string]proxyType{}
 			if labelSelector != "" {
 				if podNames, podNamespace, err = getPodNameBySelector(labelSelector); err != nil {
 					return err
 				}
-				for _, pod := range podNames {
-					name, err = setupEnvoyLogConfig("", pod, podNamespace)
-					loggerNames = append(loggerNames, name)
-				}
-				if err != nil {
-					return err
-				}
 			} else {
 				if podNames, podNamespace, err = getPodNames(args[0]); err != nil {
 					return err
 				}
-				for _, podName := range podNames {
-					name, err := setupEnvoyLogConfig("", podName, podNamespace)
-					loggerNames = append(loggerNames, name)
-					if err != nil {
-						return err
-					}
+			}
+			for _, pod := range podNames {
+				name, err = setupEnvoyLogConfig("", pod, podNamespace)
+				if err != nil {
+					return err
+				}
+				if isZtunnelPod(pod) {
+					loggerNames[name] = Ztunnel
+				} else {
+					loggerNames[name] = Envoy
 				}
 			}
+			if err != nil {
+				return err
+			}
```

Comment:
```suggestion

```

Does this do anything here? What error is it catching?  Or am I just misreading.

Reply:
It's not needed. Thanks